_data/domains.yml

```diff
@@ -476,6 +476,13 @@ websites:
       tfa: Yes
       software: Yes
       doc: https://uniregistry.help/my-account-en/what-is-two-step-verification/
+      
+    - name: United Domains USA
+      url: https://www.uniteddomains.com/
+      img: uniteddomains-usa.png
+      tfa: Yes
+      software: Yes
+      doc: https://www.uniteddomains.com/twofactorauth/
```

Comment:
Is there a better link that actually explains how to enable 2fa?

Reply:
I think this is a better link to use https://help.uniteddomains.com/hc/en-us/articles/115001458813-How-to-Enable-Disable-Two-Factor-Authentication